apps/nextjs-example/components/AppContext.tsx

```diff
@@ -7,6 +7,9 @@ import { SpikaWallet } from "@spika/aptos-plugin";
 import { AptosWalletAdapterProvider } from "@aptos-labs/wallet-adapter-react";
 import { FewchaWallet } from "fewcha-plugin-wallet-adapter";
 import { MSafeWalletAdapter } from "msafe-plugin-wallet-adapter";
+import { BloctoWallet } from "@blocto/aptos-wallet-adapter-plugin";
+import { NetworkName } from "@aptos-labs/wallet-adapter-core"
```

Comment:
let's try keep the `core` package "hidden" from dapps, if we need anything from the package - especially types - we can have the react provider exports it for us. like what we do with `WalletReadyState`
https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/apps/nextjs-example/components/WalletButtons.tsx#L1
https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/packages/wallet-adapter-react/src/useWallet.tsx#L15

Reply:
Updated!